crates/bevy_sprite/src/render/mod.rs

```diff
@@ -388,26 +404,26 @@ pub fn queue_sprites(
         let mut index = 0;
         let mut colored_index = 0;
 
-        // FIXME: VisibleEntities is ignored
-        for mut transparent_phase in &mut views {
-            let extracted_sprites = &mut extracted_sprites.sprites;
-            let image_bind_groups = &mut *image_bind_groups;
+        let extracted_sprites = &mut extracted_sprites.sprites;
+        // Sort sprites by z for correct transparency and then by handle to improve batching
```

Comment:
```suggestion
        // Sort sprites by z for correct transparency and then by handle to improve batching
        // NOTE: This can be done independent of views by reasonably assuming that all 2D views look along the negative-z axis in world space
```